mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -478,3 +487,22 @@
          :provider_identity
          {:target "GROUP"
           :provider_id provider-guid}))
+
+(defn grant-permitted?
```

Comment:
Well, if it makes you feel any better, I'll be using these in the services tests ... so I would have had to move it anyway :-;